applications/form_specs.py

```diff
@@ -345,6 +345,15 @@
         rubric=snippet("service-evaluation-audit-rubric"),
         footer=snippet("service-evaluation-audit-footer"),
         fieldsets=[
+            Fieldset(
+                label="",
+                fields=[
+                    Field(
+                        name="is_member_of_bennett_or_lshtm",
+                        label="I am a member of the Bennett Institute/LSHTM conducting a study conforming to the Service Restoration Observatory and Aftershocks workstream",
```

Comment:
double-checking - `Service Restoration Observatory and Aftershocks workstream` - should this be `Service Restoration Observatory or Aftershocks workstream`?

Reply:
Looking at the page (body updated with screenshot for context!), I think `and` is correct here because of the previous paragraph.